src/oneD/MultiNewton.cpp

```diff
@@ -205,13 +232,14 @@ int MultiNewton::dampStep(const double* x0, const double* step0,
 {
     // write header
     if (loglevel > 0 && writetitle) {
-        writelog("\n\nDamped Newton iteration:\n");
-        writeline('-', 65, false);
-
-        writelog("\n{}  {:>9s}   {:>9s}     {:>9s}   {:>9s}   {:>9s}  {:>5s} {:>5s}\n",
-                "m","F_damp","F_bound","log10(ss)",
-                "log10(s0)","log10(s1)","N_jac","Age");
-        writeline('-', 65);
+        string header = fmt::format("  {:->23}Damped Newton iteration{:->24}", "", "");
+        string separator = fmt::format("  {:->70}", "");
+        writelog("\n\n{}", header);
+
+        writelog("\n  {:<4s}  {:<10s}   {:<10s}  {:<7s}  {:<7s}  {:<7s}  {:<5s}  {:<3s}\n",
+                 "Iter", "F_damp", "F_bound", "log(ss)",
+                "log(s0)", "log(s1)", "N_jac", "Age");
+        writelog(separator);
```

Comment:
A couple of simplifications can be made here, I think:
```suggestion
        writelog("\n\n{:-^70}", " Damped Newton iteration ");
        writelog("\n  {:<4s}  {:<10s}   {:<10s}  {:<7s}  {:<7s}  {:<7s}  {:<5s}  {:<3s}\n",
                 "Iter", "F_damp", "F_bound", "log(ss)",
                "log(s0)", "log(s1)", "N_jac", "Age");
        writelog("  {:->70}", "");
```